modules/jpa-basic/pom.xml

```diff
@@ -59,11 +59,6 @@
       <artifactId>devon4j-test</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>com.devonfw.java.modules</groupId>
-      <artifactId>devon4j-configuration</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
ouch. That was a bug I introduced with PR #83 
Interesting that travis and our batch archetype IT test did not reveal it.
Thanks for fixing.